### PR TITLE
[auto] Update benchmark results

### PR DESCRIPTION
**Pull request by scottgigante**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

**Review by scottgigante-immunai — APPROVED** [review]

File: data/results/batch_integration_embed/immune_batch.json [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+            "Year": 2007, 
+            "Library": "https://scanpy.readthedocs.io/en/stable/api/scanpy.pp.combat.html", 
+            "Implementation": "https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/_batch_integration/batch_integration_graph/methods/combat.py", 
+            "Version": "1.9.1", 
+            "Runtime (min)": 5.966666666666667, 
+            "CPU (%)": 1223.9, 
+            "Memory (GB)": 23.6, 
+            "Rank": 7, 
+            "Cell Cycle Score": 1.1485731501654854, 
+            "kBET": NaN,
```
Comment (scottgigante-immunai) [inline data/results/batch_integration_embed/immune_batch.json:157]: @LuckyMD @danielStrobl these NaNs are causing deploy errors. Thoughts on what we should do about them?

Comment (LuckyMD) [reply data/results/batch_integration_embed/immune_batch.json:157]: Okay... PCR nan's I don't know why they're there yet... kBET nan's I think I found out. If `library(kBET)` doesn't work, then `np.nan` is returned. This is something we didn't notice initially, and I just started an issue (https://github.com/theislab/scib/issues/349) on it. It seems that kBET is not added as a requirement to `openproblems-r-extras`, but it didn't complain here as it just returned `np.nan` instead.

Comment (LuckyMD) [reply data/results/batch_integration_embed/immune_batch.json:157]: Do we have any install_github functionality for r installs? kBET would just need to be added via:
`install_github('theislab/kBET')`

We could also just remove the metric for now as it wasn't properly added

**Comment by LuckyMD** [comment]
PC regression only fails on scaled ComBat outputs. I'm not sure what's going on there... ComBat should remove the linear batch effect perfectly by design. This also works in our paper (https://theislab.github.io/scib-reproducibility/dataset_pancreas.html#22_Batch_correction) for this exact dataset...

**Pull request closed without merging**